src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -313,6 +313,7 @@ public async Task AddNode()
             if (assignedCores <= 0)
                 throw new ArgumentException("Assigned cores must be greater than 0!");
 
+            nodeUrl = nodeUrl.Trim();
```

Comment:
@ppekrol I would suggest applying a full validation of the URL using something like `Uri.IsWellFormedUriString()` to see if there is anything else wrong with it other than just white space.

Reply:
Done here: https://github.com/ravendb/ravendb/pull/9801